client/src/modules/journal/journal.js

```diff
@@ -313,11 +313,12 @@ function JournalController(Journal, Sorting, Grouping, Filtering, Columns, Confi
         load(vm.filters);
       })
       .catch(function (error) {
-        if(error === 'POSTING_JOURNAL.ERRORS.UNBALANCED_TRANSACTIONS'){
-          Notify.warn(error);
-        } else {
+        if(error.status){
```

Comment:
You've mixed `if(` and `if (` in the PR a few times.  ESLint prefers that you use `if (` to distinguish it from a function call.  Could you use `if (` instead of `if(`?